From 07e5cf0c416040c705b9e0454667e4334622fa4a Mon Sep 17 00:00:00 2001
From: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
Date: Fri, 21 Oct 2022 15:19:53 +0800
Subject: [PATCH 04/36] arm64: imx_v8_defconfig: Enable the generic software
 mailbox

Upstream-Status: Pending

Signed-off-by: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
---
 arch/arm64/configs/imx_v8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index 31eabe2708ea..08d21b0fe28e 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -1219,3 +1219,4 @@ CONFIG_MEMTEST=y
 CONFIG_VHOST_NET=m
 CONFIG_VIDEO_MAX9286=y
 CONFIG_VIDEO_RDACM20=y
+CONFIG_GENERIC_SOFTWARE_MAILBOX=y
-- 
2.34.1

