From e6da9c95a42d61fe5208eb73517c9839e176e759 Mon Sep 17 00:00:00 2001
From: Stephane Viau <stephane.viau@nxp.com>
Date: Mon, 3 Jan 2022 11:11:33 +0000
Subject: [PATCH 17/35] HRPN-232 configs: Fix copyright

Signed-off-by: Stephane Viau <stephane.viau@nxp.com>
---
 configs/arm64/cell-create.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/arm64/cell-create.h b/configs/arm64/cell-create.h
index 066f9407..1473a06b 100644
--- a/configs/arm64/cell-create.h
+++ b/configs/arm64/cell-create.h
@@ -1,10 +1,11 @@
 /*
  * Cell Configuration - Structure definition
  *
+ * Copyright (c) Siemens AG, 2014-2017
  * Copyright 2021 NXP
  *
  * Authors:
- *  Stephane Viau <stephane.viau@nxp.com>
+ *  Jan Kiszka <jan.kiszka@siemens.com>
  *
  * This work is licensed under the terms of the GNU GPL, version 2.  See
  * the COPYING file in the top-level directory.
-- 
2.25.1

