From 0bb051444f568ec927761d5eecae3474b713d75e Mon Sep 17 00:00:00 2001
From: Marouen Ghodhbane <marouen.ghodhbane@nxp.com>
Date: Fri, 19 May 2023 11:32:22 +0200
Subject: [PATCH 50/54] arm64: imx93-harpoon-rtos: add tpm timer devices

Signed-off-by: Marouen Ghodhbane <marouen.ghodhbane@nxp.com>
---
 configs/arm64/imx93-harpoon-rtos.h | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/configs/arm64/imx93-harpoon-rtos.h b/configs/arm64/imx93-harpoon-rtos.h
index d4d1ef92..b45a6d55 100644
--- a/configs/arm64/imx93-harpoon-rtos.h
+++ b/configs/arm64/imx93-harpoon-rtos.h
@@ -13,6 +13,8 @@
 typedef enum IRQn {
   /* Device specific interrupts */
   LPUART2_IRQn                 = 52,               /**< Low Power UART 2 */
+  TPM2_IRQn                    = 69,               /**< Timer PWM Module (TPM) 2 */
+  TPM4_IRQn                    = 108,               /**< Timer PWM Module (TPM) 4 */
 } IRQn_Type;
 
 #define RTOS_COMMON_MEMORY_REGIONS \
@@ -25,7 +27,9 @@ typedef enum IRQn {
 	MMIO_REGION_RWS(0x443c0000, 0x443c0000, KB(64)), /* IOMUXC */               \
 	MMIO_REGION_RWS(0x44450000, 0x44450000, KB(64)), /* CCM */                  \
 	MMIO_REGION_ROS(0x44480000, 0x44480000, KB(64)), /* ANA_PLL */              \
-	MMIO_REGION_RW( 0x44390000, 0x44390000, KB(64))  /* LPUART2 */
+	MMIO_REGION_RW( 0x44390000, 0x44390000, KB(64)),  /* LPUART2 */             \
+	MMIO_REGION_RW( 0x44320000, 0x44320000, KB(64)),  /* TPM2 */                \
+	MMIO_REGION_RW( 0x424f0000, 0x424f0000, KB(64))   /* TPM4 */
 
 #define RTOS_MEMRAM_MEMORY_REGIONS \
 	MEM_REGION_RWXL(0xd0000000, 0xd0000000, MB(16)), /* RAM */    \
@@ -47,7 +51,7 @@ typedef enum IRQn {
 
 /* interrupts 64..95 */
 #define RTOS_COMMON_IRQCHIP0_BITMAP2 \
-	0
+	IRQnMAP(TPM2_IRQn)
 
 #define RTOS_AUDIO_IRQCHIP0_BITMAP2 \
 	0
@@ -58,7 +62,7 @@ typedef enum IRQn {
 
 /* interrupts 96..127 */
 #define RTOS_COMMON_IRQCHIP0_BITMAP3 \
-	0
+	IRQnMAP(TPM4_IRQn)
 
 #define RTOS_AUDIO_IRQCHIP0_BITMAP3 \
 	0
-- 
2.34.1

